Source/Vima.MediaSorter/Program.cs

```diff
@@ -3,33 +3,42 @@
 using System.Globalization;
 using System.IO;
 using System.Text;
+using System.Text.RegularExpressions;
 using ImageMagick;
 
 namespace Vima.MediaSorter
 {
     public class Program
     {
-        public static readonly List<string> ImageExtensions = new List<string> {".JPG", ".JPE", ".BMP", ".GIF", ".PNG"};
+        public static readonly List<string> ImageExtensions = new List<string> { ".JPG", ".JPE", ".BMP", ".GIF", ".PNG" };
+        public static readonly List<string> VideoExtensions = new List<string> { ".MP4", ".WMV", ".FLV", ".WAV", ".M4A" };
```

Comment:
I would just leave ".MP4" for now, since this is the one that our phones currently use. We can add other ones later.